rdmo/accounts/views.py

```diff
@@ -48,7 +48,7 @@ def remove_user(request):
         log.info('Remove user form is disabled in settings PROFILE_DELETE')
         return render(request, 'profile/profile_remove_closed.html')
     form = RemoveForm(request.POST or None, request=request)
-    log.debug('Remove user form initialized for "%s"' % request.user.username)
+    log.debug(f'Remove user form initialized for "{request.user.username}"')
```

Comment:
I don't think ruff knows, that this is a logger. Usually f-strings are not used in log messages for performance.

Reply:
The f-string will format the string, even if it is not logged.
Ref: https://docs.python.org/3/howto/logging.html#optimization

Reply:
Ref: https://docs.astral.sh/ruff/rules/logging-f-string/

Reply:
yes, we had already short discussion about this.
Do you think its worth to keep the log lines out of this autofix for performance reasons? Maybe with the GH Actions it will have an impact on the speed/time?
I read this one https://betterstack.com/community/guides/logging/python/python-logging-best-practices/#5-vs-f-strings-for-string-formatting-in-logs and thought f-strings would be fine.

Reply:
thanks, didnt see that one https://docs.astral.sh/ruff/rules/logging-f-string/ . 
So this rule "logging-f-string (G004)" should also be regarded by ruff

Reply:
I'll add this then to the `tool.ruff.lint` `select` list right?
`  "G004", # logging-f-string`

Reply:
- [ ] I need to fix some of the ` "G004"` error first manually. Think in my other PRs I used f-strings ;)

Reply:
Ive updated it now and added the `G004` rule, all checks passed

Reply:
Maybe you could enable `G`?

Reply:
ok, ill try with all of the `G`s

Reply:
alright, the Gs are there and all tests passed